core/monitor/verify.go

```diff
@@ -91,19 +91,15 @@ func (e *ErrList) Proto() []*statuspb.Status {
 }
 
 // VerifyEpoch verifies that epoch is correctly signed and included in the append only log.
```

Comment:
You should probably explain what the new param `trusted` is for, and why it might differ from `epoch.GetLogRoot()`

Reply:
done